client/app/components/queries/ScheduleDialog.js

```diff
@@ -0,0 +1,258 @@
+import { react2angular } from 'react2angular';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Modal } from 'react-bootstrap';
```

Comment:
We're standardizing on using Ant's components instead of Bootstrap (already using them for date inputs and in @washort's pull requests). In this case you should probably use [Ant's Modal](https://ant.design/components/modal/).